src/Service/Helper/HelpMessageHelper.php

```diff
@@ -58,17 +53,8 @@ public function getMessage($route_name) {
         $documentation_link_text = t('Documentation');
         $documentation_link_url = Url::fromUri('https://docs.acquia.com/lift/', $link_attributes);
         $documentation_external_link = $this->linkGenerator->generate($documentation_link_text, $documentation_link_url);
-        $help_message = t('You can find more info in ') . $documentation_external_link;
-
-        // Generate Acquia Lift Web Admin link.
-        if (!empty($this->credentialSettings['api_url'])) {
-          $lift_web_link_text = t('Acquia Lift Web Admin');
-          $lift_web_link_url = Url::fromUri('https://' . $this->credentialSettings['api_url'], $link_attributes);
-          $lift_web_external_link = $this->linkGenerator->generate($lift_web_link_text, $lift_web_link_url);
-          $help_message .= t(', and control your web services settings at ') . $lift_web_external_link;
-        }
+        $help_message = t('You can find more info in ') . $documentation_external_link . t('.');
```

Comment:
It's a good sanity check. The result page reads as follows:

<img width="428" alt="screen shot 2016-07-05 at 16 26 52" src="https://cloud.githubusercontent.com/assets/1379691/16598764/68bd1d9c-42cd-11e6-9cd4-1bb10e1dfe48.png">

Which I think "in" is the correct use here.